lib/bolt/cli.rb

```diff
@@ -135,6 +135,10 @@ def create_option_parser(results)
           end
         end
         results[:concurrency] = 100
+        opts.on('-i', '--private-key KEY',
```

Comment:
lets just expose this as --private-key for now.

Reply:
OK, I am going to delete the "-i" (and associated tests), squash my new commit and push again on my fork repo.